examples/cs-schedules/MyStack.cs

```diff
@@ -0,0 +1,92 @@
+using Pulumi;
+using Pulumi.PulumiService;
+using Pulumi.PulumiService.Inputs;
+using Pulumi.Random;
+
+class MyStack : Stack
+{
+    public MyStack()
+    {
+        // Deployment Settings are required to be setup before schedules can be
+        // Note the `DependsOn` option in all of the schedules
+        var settings = new DeploymentSettings(
+            "Deployment Settings",
+            new DeploymentSettingsArgs{
+                Organization = "service-provider-test-org",
+                Project = "cs-schedules",
+                Stack = "test-stack",
+                SourceContext = new DeploymentSettingsSourceContextArgs{
+                    Git = new DeploymentSettingsGitSourceArgs{
+                        RepoUrl = "https://github.com/example.git",
+                        Branch = "refs/heads/main"
+                    }
+                }
+            }
+        );
+
+        // Schedule that runs drift every Sunday midnight, but does NOT remediate it
+        var drift = new DriftSchedule(
+            "drift-schedule",
+            new DriftScheduleArgs
+            {
+                Organization = "service-provider-test-org",
+                Project = "cs-schedules",
+                Stack = "test-stack",
```

Comment:
You can use the output from deployment settings as an input here to create the dependency chain to ensure that the schedules are always created *after* the settings.
```suggestion
                Organization = settings.Organization,
                Project = settings.Project,
                Stack = settings.Stack,
```

Reply:
Yeah, I used dependsOn below to make dependency explicit, and mentioned it above in comment to explain to the reader.

I can do the settings properties too, agree that it's cleaner. However, in this case, since it's an example I think explicitly showing dependency is better, but can change, don't mind either way

Reply:
Ah whoops I see it now. Fine to leave as-is.